### PR TITLE
Added enhanced guacamole.

### PR DESCRIPTION
**Pull request by bytebunny**
I added enhancement to the guacamole that goes in tacos.

### CODE REVIEW

**Comment by jenswehnerteaching** [comment]
@bytebunny  Thanks does this fully close https://github.com/jenswehnerteaching/forking/issues/2

File: guacamole_with_tomatoes.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+## Enhanced guacamole
+- Add tomatoes to gucamole.
```
Comment (jenswehnerteaching) [inline guacamole_with_tomatoes.md:2]:
```suggestion
- Add tomatoes to guacamole.
```

**Comment by bytebunny** [comment]
> @bytebunny Thanks does this fully close #2 ?

It does.

**Review by jenswehnerteaching — CHANGES_REQUESTED** [review]
Small spelling mistake. Otherwise really nice catch. 

Thanks

**Review by jenswehnerteaching — APPROVED** [review]
Thx a lot

**Comment by jenswehnerteaching** [comment]
closes #2

**Pull request merged**